lib/cli/src/Cardano/CLI.hs

```diff
@@ -106,10 +109,39 @@ instance FromText (Port tag) where
 instance ToText (Port tag) where
     toText (Port p) = toText p
 
+instance FromText (OptionValue Severity) where
+    fromText t = case T.toLower t of
+        "alert" ->
+            mk Alert
+        "critical" ->
+            mk Critical
+        "debug" ->
+            mk Debug
+        "emergency" ->
+            mk Emergency
+        "error" ->
+            mk Error
+        "info" ->
+            mk Info
+        "notice" ->
+            mk Notice
+        "warning" ->
+            mk Warning
+        _ -> Left $ TextDecodingError $ show (T.unpack t)
+            <> " is not a recognized log severity level."
```

Comment:
What about mentioning the expected values in the error message. It makes much more user-friendly errors. "log severity level is neither alert, criticial, debug .... nor warning (maybe ordering them by severity!)

Reply:
That's a sensible suggestion! I've added a task to this PR.

Reply:
@KtorZ I've added a patch to enumerate the complete list of severity levels within the error message. They're listed in ascending order of severity.